tests/lax_scipy_special_functions_test.py

```diff
@@ -25,9 +25,12 @@
 import jax
 from jax._src import test_util as jtu
 from jax.scipy import special as lsp_special
+from jax import config
+config.update("jax_enable_x64", True)
 
 from jax import config
 config.parse_flags_with_absl()
+config.update("jax_enable_x64", True)
```

Comment:
We should not enable x64 here; this will override the environment settings that we use to run tests both with and without x64 enabled.